x/ibc/core/04-channel/keeper/packet.go

```diff
@@ -211,11 +216,16 @@ func (k Keeper) RecvPacket(
 		)
 	}
 
+	commitment, err := types.CommitPacket(k.cdc, packet)
+	if err != nil {
+		return err
```

Comment:
nit: SendPacket/RecvPacket/AcknowledgePacket should return the same error format